src/docs/admin/references/NodeSourceInfrastructureReference.adoc

```diff
@@ -362,6 +362,8 @@ However, it implements a different instance management strategy that reduces the
 	2. The nodes share the same networking infrastructure through a common Virtual Private Cloud (VPC).
  The infrastructure supports networking autoconfiguration if no parameter is supplied.
 
+WARNING: To fully benefit from the latter management strategy, the infrastructure should be coupled with other than the Empty Policy.
```

Comment:
What does "the latter management strategy" refers to? And I feel a bit confused about this warning. I don't think we have infrastructures only work with empty policy?

Reply:
Ah, ok. I get it now. Could it be simplified as something like: `WARNING: The node source using empty policy will not benefit from this latter management strategy. The deployment with empty policy doesn't use the shared instance template and VPC.`?

Reply:
I will use this instead. 
"WARNING: The node source using empty policy will not benefit from this latter management strategy. The deployment with empty policy doesn't use the shared instance template and networking configuration."
I prefer `networking configuration` instead of vpc as it is wider (vpc, subnet and sg).

Reply:
Some small edits but otherwise LGTM